doc/topics/grains/index.rst

```diff
@@ -132,6 +132,15 @@ Using Jinja templating, only one match entry needs to be defined.
 Writing Grains
 ==============
 
+Consider adding Gains when the information is needed for targeting,
```

Comment:
Misspelling and unnecessary capitalization. Also this sentence needs to be improved, or split into two sentences. As one sentence, it doesn't make much sense.